ui/src/app/base/components/VLANSelect/VLANSelect.tsx

```diff
@@ -12,15 +12,17 @@ import { getVLANDisplay } from "app/store/vlan/utils";
 
 type Props = {
   defaultOption?: { label: string; value: string } | null;
+  filterFunction?: (subnet: VLAN) => boolean;
```

Comment:
probs should be `vlan` instead of `subnet`

Reply:
Hah, thanks!